commandeer.nim

```diff
@@ -270,19 +270,42 @@ template errormsg*(msg: string): untyped =
   errorMessage = msg
 
 
-template subcommand*(identifier: untyped, subcommandName: string,
-                    statements: untyped): untyped =
-    var identifier: bool = false
-    subCommands[subcommandName] = CommandLineMapping(
-      assigners: newSeq[Assigner](),
-      shortOptions: newTable[string, assignmentProc](),
-      longOptions: newTable[string, assignmentProc](),
-      activate: proc() =
-        identifier = true
-    )
-    currentMapping = subCommands[subcommandName]
-    statements
-    currentMapping = subCommands[""]
+#[#old implementation of subcommand
+  
+  template subcommand*(identifier: untyped, subcommandName: string,
+                     statements: untyped): untyped =
+  var identifier: bool = false
+  subCommands[subcommandName] = CommandLineMapping(
+    assigners: newSeq[Assigner](),
+    shortOptions: newTable[string, assignmentProc](),
+    longOptions: newTable[string, assignmentProc](),
+    activate: proc() =
+      identifier = true
+  )
+  currentMapping = subCommands[subcommandName]
+  statements
+  currentMapping = subCommands[""]
+  ]#
```

Comment:
Very cool that only one template makes it work. A simpler test with only one template wasn't working for me but this is. Strange, but I will take it!